seq2science/rules/quantification.smk

```diff
@@ -307,6 +307,12 @@ elif config["quantifier"] == "kallistobus":
                 -t {threads} -g {params.basename}_t2g.txt \
                 -o {output} -c1 {params.basename}_cdna_t2c.txt -c2 {params.basename}_intron_t2c.txt \
                 {params.barcode_arg} {params.options} {input.reads} > {log} 2>&1
+                # Validate output
+                if grep -q 'ERROR\|bad_alloc' "{log}"; then
+                  echo "An error was encoutered. Please check the logfile: {log} for details!" 
+                  if [ -e "{output}" ]; then rm -Rf {output}; fi
+                  exit 1
```

Comment:
I think you can just `exit 1`, snakemake will remove the file for you

Reply:
Nice, didn't know that :)